.github/workflows/call-run-tests.yml

```diff
@@ -14,14 +14,20 @@ jobs:
           node-version: 20
           cache: 'npm'
 
+      - name: Install jq
+        run: sudo apt-get install jq -y
+
       - name: Install dependencies
         run: npm ci
 
       - name: Install Playwright Browsers
         run: npx playwright install --with-deps
 
-      - name: Run Playwright tests
-        run: ./run-tests.sh
+      - name: Run Playwright short tests
+        run: ./run-tests.sh -t short
+
+      - name: Run Playwright long tests
+        run: ./run-tests.sh -t long
```

Comment:
A short comment on options would be helpful here
`# run_tests.sh required flag "type" -t [short|long], optional flag "browser"-b [firefox|chrome|...]`

Question: can there be more than one browser set in `-b` Please document so future devs can most easily know without having to read code. Thanks!